clippy_lints/src/loops/mod.rs

```diff
@@ -188,22 +188,22 @@ declare_clippy_lint! {
     /// The `while let` loop is usually shorter and more
     /// readable.
     ///
-    /// ### Known problems
```

Comment:
Referred issue was closed long time ago, so remove it, this this lint has some issues which can be found  at section [Related Issues](https://github.com/rust-lang/rust-clippy/issues?q=is%3Aissue+while_let_loop)